core/templates/domain/exploration/InteractionObjectFactory.ts

```diff
@@ -486,7 +486,7 @@ export class InteractionObjectFactory {
 
   createAnswerGroupsFromBackendDict(
       answerGroupBackendDicts: readonly AnswerGroupBackendDict[],
-      interactionId: string | null
+      interactionId: string | null,
```

Comment:
Similar question here.

Reply:
https://github.com/oppia/oppia/blob/ac68b404fef555b6982613effc483261365ea147/core/templates/domain/state/StateObjectFactory.ts#L148
This method is called by stateObjectFactory, and interactionId is null in default state.